src/utils/api.js

```diff
@@ -85,13 +85,18 @@ const callAPI = (route, options, rawError = false) => {
       }
 
       if (constants.DEBUG || global.argOpts.debug) {
-        console.log(`>> ${requestOptions.method} ${requestOptions.url}`);
+        console.log(`\n>> ${requestOptions.method} ${requestOptions.url}`);
         if (requestOptions.body) {
           const replacementStr = 'raw zip removed in logs';
-          let cleanedBody = _.assign({}, JSON.parse(requestOptions.body), {
-            zip_file: replacementStr,
-            source_zip_file: replacementStr
-          });
+          const requestBody = JSON.parse(requestOptions.body);
+          const cleanedBody = {};
+          for (const k in requestBody) {
+            if (k.includes('zip_file')) {
```

Comment:
this prevents adding a `zip_file` key when there wasn't one before. I was super confused as to why my `DELETE` call had 2 zips attached.